net-misc/hylafaxplus/hylafaxplus-7.0.2.ebuild

```diff
@@ -0,0 +1,165 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit pam toolchain-funcs
```

Comment:
EAPI-7 should work here, + one empty line between EAPI and inherit.

Reply:
Is it appropriate to bump EAPI without sorting out depends into both {,B}DEPEND? I'm not familiar with this package as I'm just bumping it for security issues so I wasn't confident in my ability to accurately do this.

Reply:
Sure, I'll split them if I spot something obvious. Fast-glancing through didn't look like there was any.